PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcher.cpp

```diff
@@ -89,6 +89,41 @@ intptr_t CALLBACK VerticalFileSwitcher::run_dlgProc(UINT message, WPARAM wParam,
 			return TRUE;
 		}
 
+		case WM_PARENTNOTIFY:
+		{
+			switch ( wParam )
+			{
+				case WM_MBUTTONDOWN:

```

Comment:
According several browsers and several windows OS middle click to close behaviour (that I checked) - they are all closed on mous up. So here should be `WM_MBUTTONUP` instead.